runtime/src/accounts_background_service/pending_snapshot_packages.rs

```diff
@@ -113,6 +113,11 @@ impl PendingSnapshotPackages {
             (None, None) => None,
         }
     }
+
+    #[cfg(feature = "dev-context-only-utils")]
+    pub fn snapshots_pending(&self) -> bool {
```

Comment:
I was trying to think of a good name for this fn. Mostly I'd like it to convey a boolean return type. Something like "are_there_any_pending_snapshot_packages()", but without being so insanely long, and without duplicating the name of the struct itself. But I didn't really like any that I thought of. 

Then it hit me. The converse of this fn exists already: `is_empty()`! Wdyt about using that here too. Then we don't need to repeat any words for "pending" or "snapshot" or "package", and it clearly indicates boolean.

(and changing the impl of course)